docs/sources/panels/visualizations/alert-list-panel.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Alert list panel"
+keywords = ["grafana", "alert list", "documentation", "panel", "alertlist"]
+type = "docs"
+aliases = ["/docs/grafana/latest/reference/alertlist/", "/docs/grafana/latest/features/panels/alertlist/"]
+[menu.docs]
+name = "alert-list-panel"
+parent = "visualizations"
+weight = 750
+draft = "true"
++++
+
+
+# Alert list panel
+
+The alert list panel allows you to display your dashboards alerts. You can configure the list to show current state or recent state changes. You can read  more about alerts in [Alerting rules](http://docs.grafana.org/alerting/rules).
+
+{{< docs-imagebox img="/img/docs/v45/alert-list-panel.png" max-width="850px" >}}
+
+## Options
+
+Use these settings to refine your visualization.
+
+### Options
+
+- **Show -** Choose whether the panel should display the current alert state or recent alert state changes.
+- **Max Items -** Sets the maximum number of alerts to list.
+- **Sort order -** Select how to order the alerts displayed:
+  - **Alphabetical (asc) -** Alphabetical order.
+  - **Alphabetical (desc) -** Reverse alphabetical order.
+  - **Importance -** By importance.
```

Comment:
There is a ranking that is used to sort by importance:

- alerting: 1,
- no_data: 2,
- pending: 3,
- ok: 4,
- paused: 5

Maybe can mention that?

Reply:
Yes, I think this is a good idea.

Reply:
Will do